README.md

```diff
@@ -27,6 +27,24 @@ Once these are installed successfully complete the following steps to setup the
 
 That's it! Sing and dance. The email template code is ready for development and testing.
 
+## Setup and Installation with Docker
+
+You will need the following tools installed in order to complete the setup.
+
+- [Docker](https://www.docker.com)
+
+Once these are installed successfully complete the following steps to setup the project (run commands from the project root directory).
+
+1. Copy `settings.sample.json` to `settings.json` and configure the following:
+    - From email address
+    - Default To email address
+    - SMTP Provider settings (see a [full list of supported providers](https://github.com/andris9/nodemailer-wellknown#supported-services))
+1. Run the command `docker-compose up` to build and run the docker image.
```

Comment:
Could you add an additional note for those new to Docker on how to tear down the docker image (`docker-compose down`)?

Reply:
I've added an additional note. Let me know if you would like anything else.